pandas/tests/indexes/common.py

```diff
@@ -933,3 +933,10 @@ def test_empty(self):
         index = self.create_index()
         assert not index.empty
         assert index[:0].empty
+
+    @pytest.mark.parametrize('how', ['outer', 'inner', 'left', 'right'])
```

Comment:
I mean this test here: https://github.com/pandas-dev/pandas/issues/16541#issue-232356015